app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -135,7 +135,7 @@ angular.module("openshiftConsole")
            scope.storageClasses = _.sortBy(storageClasses, 'metadata.name');
            var annotation = $filter('annotation');
            scope.defaultStorageClass = _.find(scope.storageClasses, function(storageClass) {
-             return annotation(storageClass, 'storageclass.beta.kubernetes.io/is-default-class') === 'true';
+             return annotation(storageClass, 'storageclass.kubernetes.io/is-default-class') === 'true';
```

Comment:
If I recall correctly, we can't assume the old `annotation` goes away, however (it will linger in use).  Perhaps we could update to check both...

Reply:
Agree, we'd ideally check for either annotation.